tests/tasks/setup/eks/awscli-cp-with-vpc.yaml

```diff
@@ -27,9 +27,14 @@ spec:
   - name: aws-ebs-csi-driver-version
     default: release-1.13
     description: The release version for aws ebs csi driver.
+  workspaces:
+  - name: config
+    mountPath: /config/
+    description: |
+      A workspace into which a kubeconfig file called `kubeconfig` will be written that will contain the information required to access the cluster. The `kubeconfig` will expect to use [aws-iam-authenticator](https://github.com/kubernetes-sigs/aws-iam-authenticator/) to authenticate, so in order for it to be used it must be run in a container which contains both `kubectl` and `aws-iam-authenticator`.
   steps:
   - name: create-eks
-    image: alpine/k8s:1.23.13
+    image: alpine/k8s:1.23.7
```

Comment:
why are we going back in version ? 
Did you pick the older commit we had with this `kubeconfig` change ?

Reply:
the reason is to keep all the alpine k8s images common among all task. I have observed some kubectl api calls breaking when I used 1.23.13 due v1alpha1 to v1beta1 transition of some resource when i was testing it with deveks. I have tried with 1.23.7 and seems like its pretty stable. We can upgrade all the images together as we iterate on the versions. wdyt ?

Reply:
Sure. Sounds good. Let's make sure that image is consistent across all the tasks.